firebase_config.js

```diff
@@ -0,0 +1,28 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import.meta.env;
+
+const apiKey = import.meta.env.VITE_apiKey;
+const authDomain = import.meta.env.VITE_authDomain;
+const projectId = import.meta.env.VITE_projectId;
+const storageBucket = import.meta.env.VITE_storageBucket;
+const messagingSenderId = import.meta.env.VITE_messagingSenderId;
+const appId = import.meta.env.VITE_appId;
+
```

Comment:
it might be helpful to add some error handling to make sure the environment variables are properly set in the environment. You can add this here : 
`if (!apiKey || !authDomain || !projectId || !storageBucket || !messagingSenderId || !appId) {
  throw new Error("Firebase environment variables are not set correctly.");
}
`
This way if the firebase initialization fails, we will be alerted.